### ISSUE TITLE
Use-case for manual `expanded` control

### ISSUE THREAD

**Issue by ntucakovic** [springload/react-accessible-accordion#241]
I have a scenario where I've got an anchor which links to an accordion item which is below the fold.

When user clicks on the link, he's scrolled down to the accordion item.
I'd prefer to have it opened once it's in viewport rather than having to click again to expand to see content.

And to do that elegantly I need an API which allows to expand/collapse desired accordion items.

API to do that could look like:
```tsx
const [accordion, setAccordion] = useState(null);

const handleAccordionItemExpand = useCallback((itemId) => {
    if (accordion) {
        accordion.expand(itemId);
    }
}, [accordion]);

return (
    <a href="#reviews" onClick={handleAccordionItemExpand}>Read reviews</a>

    ...

    <Accordion setRef={setAccordion}>
      ...
    </Accordion>
)
```

Is there any other way to do this except targeting DOM to trigger click events on the accordion button?

> If you believe that you have a valid use-case for 'disabled' items, or items which require manual 'expanded' state-management, then please let us know - we're always open for critical (but polite) feedback. Otherwise, we don't plan on implementing this functionality in the near future.

**Comment by yuzima** [comment]
Can you update `preExpanded` to expand the item I want when user click the link.

**Comment by synecdokey** [comment]
`dangerouslySetExpanded` is now available as an `AccordionItem` prop in version 3.1.0 ✨ This should neatly solve the issue!
